Project build fails on clean CentOS 8 host

Created a clean VM from CentOS 8 stream (server with GUI).

Forked repo and checked out main branch.

Attempting to run `make verify` got this error:

~~~
Running `gofmt -s -l` on 584 file(s).
go vet -mod=vendor ./...
# github.com/mtrmac/gpgme
vendor/github.com/mtrmac/gpgme/data.go:4:11: fatal error: gpgme.h: No such file or directory
 // #include <gpgme.h>
           ^~~~~~~~~
compilation terminated.
make: *** [vendor/github.com/openshift/build-machinery-go/make/targets/golang/verify-update.mk:26: verify-govet] Error 2
~~~

--- a/README.md
+++ b/README.md
@@ -28,6 +28,13 @@ workstation, install them with:
 dnf install krb5-devel
 ```
 
+Also:
+
+```
+dnf install gpgme-devel
+dnf install libassuan-devel
+```
+
 For MacOS you'll need to install a few brew packages before building locally. Install them with:
 ```
 brew install heimdal

